.travis.yml

```diff
@@ -4,4 +4,12 @@ services:
     - docker
   
 script:
-    - ./test-build.sh
+    - ./test-build.sh $NODE_VERSION
```

Comment:
I think I prefer reverting this so that `test-build.sh` automatically runs against all node versions it finds rather than having the node versions hard-coded in the env matrix. It's one more file we'd need to update for every major/minor Node.js release and forgetting to do that is an easy mistake to make (like with `generate-stackbrew-library.sh`).